src/Bridge/Ramsey/Normalizer/UuidNormalizer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Ramsey\Normalizer;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+/**
+ * Denormalizes an UUID string to an instance of Ramsey\Uuid.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class UuidNormalizer implements DenormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function denormalize($data, $class, $format = null, array $context = [])
+    {
+        return Uuid::fromString($data);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsDenormalization($data, $type, $format = null)
+    {
+        return interface_exists(UuidInterface::class) && (UuidInterface::class === $type || is_subclass_of($type, UuidInterface::class));
```

Comment:
I think you can remove the `interface_exists` call (at least, you should check `UuidInterface::class === $type` first because it's less coastly).